source/transfer_list.rst

```diff
@@ -865,12 +865,15 @@ the OP-TEE OS.
      - hdr_size
      - Holds the address of OP-TEE pageable part
 
-**DT formatted SPMC manifest entry layout (XFERLIST_DT_SPMC_MANIFEST)**
+**DT formatted FF-A manifest entry layout (XFERLIST_DT_FFA_MANIFEST)**
 
-This entry type holds the SPMC (Secure Partition Manager Core) manifest image
-which is in DT format [DT]_ and described in [TFAFFAMB]_.
-This manifest contains the SPMC attribute node consumed by the SPMD
-(Secure Partition Manager Dispatcher) at boot time.
+This entry type holds an FF-A manifest, which can either be an SPMC (Secure Partition Manager Core)
+manifest image, or an FF-A Secure Partition (SP) manifest image.
+The SPMC manifest contains the SPMC *attribute* node consumed by the SPMD (Secure Partition Manager
+Dispatcher) at boot time [SPMCATTR]_.
+The FF-A SP manifest has the partition's information following the FF-A manifest binding specification
+[TFAFFAMB]_.
```

Comment:
Given that the 2 manifests use 2 different specs, I've changed my mind about making the existing TE more generic. I now think the existing SPMC TE should have its reference fixed and then add a new one for the SP manifest.

Reply:
It's not so much about the risk of these manifests existing in the same TL, but that they represent different logical objects. It's true that a side-effect of using the DT format is that it enables easy detection of the content, but we've had strong feedback previously that we should not overload TEs that use DT format with different content. Otherwise, we could just use the existing XFERLIST_FDT for all these uses of DT formatted objects. Or even replace this spec with DT!

Actually, the preference is to not overload use of DT at all and use TEs defined in this spec if possible. That might be too disruptive in this case but hopefully gives you an idea of the direction of travel.